docs/KnownLimitations.md

```diff
@@ -0,0 +1,47 @@
+# Known Limitations
+
+List where libical doesn't fully support all the features described in the RFCs.
+
+Contributions to help us fix these limitations are welcome.
+
+## RFC 5545
+
+### Recurrence Rule section 3.3.10
+
+* BYSETPOS is only supported for MONTHLY and YEARLY frequencies (<https://github.com/libical/libical/issues/795>)
+
+* When used as a limit (rather than for expansion) negative values are not supported
+  for BYMONTHDAY, BYYEARDAY. possibly others (<https://github.com/libical/libical/issues/791>)
```

Comment:
Probably 'possibly others' is not too helpful for end users and I don't think there are others. Comment was there more for internal discussion. Would remove it.